### PR TITLE
Fixing references / view file due to view compilation errors

### PR DESCRIPTION
**Pull request by jimasp**
see: https://github.com/OrchardCMS/Orchard/issues/7713

### CODE REVIEW

File: src/Orchard.Web/Modules/Orchard.Dashboards/Web.config [hunk 1]
```diff
@@ -7,7 +7,7 @@
         </sectionGroup>
     </configSections>
     <system.web.webPages.razor>
-        <host factoryType="System.Web.Mvc.MvcWebRazorHostFactory, System.Web.Mvc, Version=5.2.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35" />
+        <host factoryType="System.Web.Mvc.MvcWebRazorHostFactory, System.Web.Mvc, Version=5.2.3, Culture=neutral, PublicKeyToken=31bf3856ad364e35" />
```
Comment (sebastienros) [inline src/Orchard.Web/Modules/Orchard.Dashboards/Web.config:10]: I assume the specflow web.config file also needs to change

File: src/Orchard.Web/Modules/Orchard.Templates/Views/DefinitionTemplates/Parts.Shape.cshtml [hunk 2]
```diff
@@ -14,13 +14,6 @@
 
 <fieldset>
     <div>
-        @Html.LabelFor(m => m.Name, T("Name"))
-        @Html.TextBoxFor(m => m.Name, new { @class = "text medium" })
-        <span class="hint">@T("The name of this template, which can be used to reference from code (as when using shape type names) and tokens.")</span>
-    </div>
-</fieldset>
-<fieldset>
-    <div>
```
Comment (BenedekFarkas) [inline src/Orchard.Web/Modules/Orchard.Templates/Views/DefinitionTemplates/Parts.Shape.cshtml:23]: I don't think removing this is correct, although I also see build errors with view compilation from this piece of code - I'll look into it.

File: src/Orchard.Web/Modules/Orchard.Dashboards/Web.config (hunk 1 above)
Comment (BenedekFarkas) [reply src/Orchard.Web/Modules/Orchard.Dashboards/Web.config:10]: @sebastienros I think that was also discovered when there was a problem with running the site with IIS Expess, but I don't remember now where we got with that.

File: src/Orchard.Web/Modules/Orchard.Templates/Views/DefinitionTemplates/Parts.Shape.cshtml (hunk 2 above)
Comment (BenedekFarkas) [reply src/Orchard.Web/Modules/Orchard.Templates/Views/DefinitionTemplates/Parts.Shape.cshtml:23]: Oh, actually this needs to be removed as Template has TitlePart!

**Comment by sebastienros** [comment]
Should we merge this one or wait for more changes from you @BenedekFarkas ?

**Comment by BenedekFarkas** [comment]
I think we can merge this, as it definitely brings us closer to the goal, but let's not close the issue yet.
I'll open another PR with my changes after this one is merged, then we can discuss how to proceed (judging from what @jimasp commented on the issue, he's testing from VS, but I think validating with MSBuild is also necessary, see my comments on the issue about that).

**Pull request merged**